docs/reference/commandline/run.md

```diff
@@ -1123,6 +1123,31 @@ $ docker run \
 hello from host!
 ```
 
+## <a name="log-driver"></a> Logging drivers (--log-driver)
+
+The container can have a different logging driver than the Docker daemon. Use
+the `--log-driver=VALUE` with the `docker run` command to configure the
+container's logging driver. The following options are supported:
+
+| Driver       | Description                                                                                                                    |
+|:-------------|:-------------------------------------------------------------------------------------------------------------------------------|
+| `none`       | Disables any logging for the container. `docker logs` won't be available with this driver.                                     |
```

Comment:
Perhaps we should remove the overview here (we could limit to describe `none` "to disable logging"). That way we can keep the logging section as the canonical overview.